init-open-recentf.el

```diff
@@ -26,22 +26,23 @@
 ;;; Commentary:
 ;;
 ;; Open recentf immediately after Emacs is started.
-;; If files are opend, does nothing.  Open recentf otherwise.
-;; (For example, it is when execute by specifying the file from command line.)
-;; This script uses only advice function for startup.  Do not have interactive function.
```

Comment:
@zonuexe I'm not sure exactly what the "advice" function was. It sounded like you were saying that the emacs chose the default/inbuilt functions of the init-open-recentf.el file, rather than using or requiring a users interaction with an interactive function.